app/src/main/java/database/migration/Migrator.kt

```diff
@@ -0,0 +1,17 @@
+package database.migration
+
+import android.app.Application
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import database.ObjectBox
+import database.migration.Migration0_1
+
+
+class Migrator: AppCompatActivity() {
```

Comment:
So how does this activity get invoked? Does it always run before the app starts, or does it run on install, or what?

Reply:
You run it manually to populate the database. I want to make it run a check on each app start, but it's out of scope of this feature at this point